src/coaching-chatbot/dialog.js

```diff
@@ -306,6 +307,7 @@ bot
           session.addQuickReplies([
```

Comment:
Here one could use `Builder.QuickReplies.createArray` instead of calling `create` multiple times.